### ISSUE TITLE
Outlook Plugin unter Windows bitte aufnhemen

### ISSUE THREAD

**Issue by Elompenta** [Jugendhackt/HowToPGP#9]
Nettes Projekt - Gefällt mir sehr gut.

Bitte nehmt doch das "OutlookPrivacyPlugin" für Outlook mit auf.
Läuft wunderbar in allen Versionen und ist mit einem Klick zu installieren.

Github Repository: https://github.com/dejavusecurity/outlookprivacyplugin

**Comment by two-heart** [comment]
Danke für den Hinweis, werde ich mir anschauen. Weißt du zufällig ob es dafür eine Dokumentation unter freier Lizenz gibt? Das würde es uns ermöglichen die Anleitung zeitnah hinzuzufügen.

**Comment by Elompenta** [comment]
Nein, die Doku in Form von Wiki habe ich erst vor kurzem in der Github-Repo-Wiki ergänzt.
Diese ist aber noch leer ;)

Wenn du ein wenig Gefühl für PGP hast, sollten die 4 verschiedenen Knöpfe aber eigentlich selbst erklärend sein. Das Plugin setzt auf Kleopatra und bestehende Schlüssel auf. Es ist lediglich die Abstrahierung für den User ins Outlook und übernimmt dir crypt vorgänge
